src/kani-compiler/src/codegen_cprover_gotoc/codegen/statement.rs

```diff
@@ -45,8 +46,18 @@ impl<'tcx> GotocCtx<'tcx> {
             TerminatorKind::SwitchInt { discr, switch_ty, targets } => {
                 self.codegen_switch_int(discr, *switch_ty, targets)
             }
-            TerminatorKind::Resume => Stmt::assert_false("resume instruction", loc),
-            TerminatorKind::Abort => Stmt::assert_false("abort instruction", loc),
+            TerminatorKind::Resume => self.codegen_assert(
```

Comment:
Perhaps add a `codegen_assert_false` to avoid having to pass `Expr::bool_false` each time.

Reply:
Agreed, will add that.